analytics/CAR-2021-01-007.yaml

```diff
@@ -0,0 +1,34 @@
+---
+title: Detecting Tampering of Windows Defender Command Prompt
+submission_date: 2020/12/11
+information_domain: 'Host'
+platforms:
+  - Windows
+subtypes:
+  - Process
+analytic_types:
+  - TTP
+contributors:
+  - Cyware Labs
+id: CAR-2021-01-007
+description: |-
+  In an attempt to avoid detection after compromising a machine, threat actors often try to disable Windows Defender. This is often done using “sc” [service control], a legitimate tool provided by Microsoft for managing services. This action interferes with event detection and may lead to a security event going undetected, thereby potentially leading to further compromise of the network.
+  - technique: T1562
+    coverage: Medium
+    subtechniques:
+      - 001
+    tactics:
+      - TA0005
+implementations:
+- name: Splunk search - Detecting Tampering of Windows Defender Command Prompt
+
+  description:  This query looks for the specific use of service control for querying or trying to stop Windows Defender.
+
+  code: |-
+    index=* EventCode=1 Image = *sc.exe* CommandLine="*stop WinDefend*" OR CommandLine="*config WinDefend*" OR CommandLine="*query WinDefend*"
+| table _time host Image CommandLine
+
+  data_model: Sysmon native
+  type: Splunk
+data_model_references:
+  - process/create/command_line
```

Comment:
Add a pseudocode implementation above, and then make sure you add all the fields you used here under data_model_references.